Packs/AWS-Enrichment-Remediation/ReleaseNotes/1_1_4.md

```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+
+##### AWSRecreateSG
+
+- Added the **assume_role** optional input in case a roleArn needs to be passed in.
+- Updated the script to work even when there are multiple AWS-EC2 integration instances configured.
```

Comment:
Think this might be a bit better
```suggestion
- Updated the script to work with multiple AWS-EC2 integration instances configured.
```

Reply:
Shirley already reviewed so I prefer to keep as-is